pkg/domain/response.go

```diff
@@ -1,31 +1,36 @@
-package handlers
+package domain
 
 type response struct {
 	Message string `json:"message"`
-	Status string `json:"status"`
+	Status  string `json:"status"`
 }
 
-func response200(msg string) response{
+func Response200(msg string) response {
 	var res response
 	res = response{Message: msg, Status: "200"}
 	return res
 }
 
-func response400(msg string) response{
+func Response400(msg string) response {
 	var res response
 	res = response{Message: msg, Status: "400"}
 	return res
 }
 
-func response404(msg string) response{
+func Response404(msg string) response {
 	var res response
 	res = response{Message: msg, Status: "404"}
 	return res
 }
 
-func response500(msg string) response{
+func Response500(msg string) response {
 	var res response
 	res = response{Message: msg, Status: "500"}
 	return res
 }
 
+func Response401(msg string) response {
+	var res response
+	res = response{Message: msg, Status: "401"}
+	return res
+}
```

Comment:
The http package has all of these statusCodes predefined. And the Gin package can handle some of these responses too.